docs-source/content/userguide/managing-domains/domain-events.md

```diff
@@ -242,6 +243,39 @@ Source:
 Type:    Warning
 Events:  <none>
 
+```
+Example of a `InvalidReplicasValue` event:
```

Comment:
change `InvalidReplicasValue` to `DomainValidationError`?

Reply:
thanks. updated with example of DomainValidationError.